src/lib.rs

```diff
@@ -110,9 +110,9 @@ impl MacAddress {
         self.eui.iter().all(|&b| b == 0xFF)
     }
 
-    /// Returns true if bit 1 of Y is 1 in address 'xY:xx:xx:xx:xx:xx'
+    /// Returns true if bit 1 of Y is 0 in address 'xY:xx:xx:xx:xx:xx'
```

Comment:
bit 0?

Reply:
Correction. IEEE EUI definitions define M bit for LSb, and X bit for MSb, and does not define the bit numbering. That said, numerous IEEE documents in fact use LSb 0 bit numbering system. (eg. http://standards.ieee.org/develop/regauth/tut/22bit.pdf)

This crate appears to use LSb 1 bit numbering system. Was it documented? It would be greater to explicitly say so.